spec/controllers/submissions_controller_spec.rb

```diff
@@ -854,13 +854,48 @@
       end
 
       describe '#manually_collect_and_begin_grading' do
-        before do
+        it('it should respond with 302') do
```

Comment:
You can remove the parentheses around the argument (this was a stylistic error with the original test). Also, the `it` string argument should not itself contain the word "it", as that's inserted automatically by rspec.